mosquitto/config.json

```diff
@@ -24,8 +24,10 @@
       "active": false,
       "folder": "mosquitto"
     },
-    "certfile": "fullchain.pem",
-    "keyfile": "privkey.pem"
+    "cafile": "cafile.pem",
```

Comment:
let it work like the old version. Don't set that option default and if they are not set, use the old hardcoded version.